os/include/tinyara/audio/audio.h

```diff
@@ -293,8 +293,8 @@
 /* Speech Detect PU controls **************************************************/
 
 #define AUDIO_SD_UNDEF              0x0000
-#define AUDIO_SD_ENDPOINT_DETECT    0x0001
-#define AUDIO_SD_KEYWORD_DETECT     0x0002
+#define AUDIO_SD_KEYWORD_DETECT     0x0001
```

Comment:
EPD is more common functionality of h/w that's why I defiend EPD first.

Reply:
Ok, I reverted.